src/content/docs/mobile-monitoring/new-relic-mobile-android/troubleshoot/sentry-incompatibility-android.mdx

```diff
@@ -0,0 +1,28 @@
+---
+title: Sentry incompatibility (Android)
+type: troubleshooting
+tags:
+  - Mobile monitoring
+  - New Relic Mobile Android
+  - Troubleshoot
+metaDescription: 'New Relic Android agent Log instrumentation cannot coexist with Sentry Android'
+freshnessValidatedDate: never
+---
+
+## Problem
+
+Conflicts, crashes, or errors will show when both the New Relic Android agent and Sentry's [Sentry Monitoring SDK](https://docs.sentry.io/platforms/android/) are included within an app.
```

Comment:
Conflicts, crashes, or errors will show when both the New Relic Android agent and [Sentry Monitoring SDK](https://docs.sentry.io/platforms/android/) are included within an app.